uint8 my_counter
bool isFull 
float32 x
float32 y
